src/styles/main.scss

```diff
@@ -1,4 +1,6 @@
 @import './base/colors.scss';
 @import './base/general.scss';
 @import './base/mixins.scss';
+@import './components/IMGs/pistolImg.scss';
+@import './components/Header.scss';
```

Comment:
`Header` could be named as `header` as there are styles not React component itself. So -> React components in JavaScript will be capitlized, components in scss not. 
Also, you have an empty folder modules -> not used right now, so let's remove it.